public_html/layouts/basic/modules/Vtiger/resources/List.js

```diff
@@ -1311,6 +1311,21 @@ jQuery.Class("Vtiger_List_Js", {
 
 		});
 	},
+	registerModalMassAction: function () {
```

Comment:
unnecessary because we already have such a mechanism only need to improve it
https://github.com/YetiForceCompany/YetiForceCRM/blob/d97336ba5afd5ca2c579e5f728c14c465231bca9/public_html/layouts/basic/modules/Vtiger/resources/List.js#L1675-L1722